script.js

```diff
@@ -0,0 +1,153 @@
+            <img src="./img/Comment Outline.svg" alt="" />
+          </div>
+        </div>
+        <div class="main__users">
+          <div class="main__user">
+            <img class="main__user-img" src="./img/user-photo.jpg" alt="" />
+            <span class="main__user-name">bruno bucciarati</span>
+            <p class="main__user-about">UX/UI designer</p>
+            <div class="main__user_social">
+              <img src="./img/hh.ru.png" alt="img" />
+              <img src="./img/twitter.png" alt="img" />
+              <img src="./img/telegram.png" alt="img" />
+            </div>
+          </div>
+          <div class="main__setting">
+            <h2 class="main__setting_nav">Navigation</h2>
+            <div class="main__setting_element">
+              <img src="./img/Person Outline.svg" alt="img" />
+              <span>My profile</span>
+            </div>
+            <div class="main__setting_element">
+              <img src="./img/Finance Outline.svg" alt="img" />
+              <span>Balance</span>
+            </div>
+            <div class="main__setting_element">
+              <img src="./img/Scheme Outline.svg" alt="img" />
+              <span>Connections</span>
+            </div>
+          </div>
+          <div class="main__adding">
+            <h3>Share your thoughts</h3>
+            <textarea placeholder="Enter your message..."></textarea>
+            <button>Save</button>
+          </div>
+        </div>
+      </section>
+        `;
+        break;
+      case "/timer":
+        app.innerHTML = `
+        <section class="main__timer">
+        <h2>Timer</h2>
+        <div id="timer">00:00:00</div>
+      </section>
+      `;
+        break;
+      case "/map":
+        app.innerHTML = `
+        <section class="main__map">
+        <h2>Basic map</h2>
+        <div id="map" style="width: 100%; height: 400px;"></div>
+      </section>
+        `
+        if(!mapInitialized) {
+          initMap()
+        }
+        ;
+        break;
+      default:
+    }
+  }
+
+  function updateHashWithoutReload(path) {
+    window.history.pushState({}, '', '#' + path);
+  }
+
+  navLinks.forEach((link) => {
+    link.addEventListener("click", function (e) {
+      e.preventDefault();
+      const newPath = this.getAttribute("href").slice(1);
+      renderContent(newPath);
+      updateHashWithoutReload(newPath);
+    });
+  });
+
+  window.addEventListener('popstate', function(event) {
+    const path = window.location.hash.slice(1);
+    renderContent(path);
+  });
+
+  const currentPath = window.location.hash.slice(1);
+  renderContent(currentPath);
+});
+
+document.addEventListener("DOMContentLoaded", function () {
+  const openMapButton = document.querySelector('.open-map');
+  openMapButton.addEventListener('click', function () {
+    initMap();
+      mapInitialized = true;
+  });
+});
+
+function initMap() {
+  var map = L.map('map').setView([55.751244, 37.618423], 10);
```

Comment:
```suggestion
  const map = L.map('map').setView([55.751244, 37.618423], 10);
```
и вообще какой var 21 век)